pypiprivate/config.py

```diff
@@ -1,6 +1,9 @@
 import os
-from ConfigParser import SafeConfigParser
 
+try:
+    from ConfigParser import SafeConfigParser
+except ModuleNotFoundError:
+    from configparser import SafeConfigParser
 
 class Config(object):
```

Comment:
Can you add one more blank line before the class definition. This will be in conformation with PEP8. 

Please refer to: 
  - Error code `E302` here - https://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes
  - https://www.python.org/dev/peps/pep-0008/#blank-lines

I will setup tests and linting on TravisCI soon for this repo